talk/objectorientation/constructors.tex

```diff
@@ -101,11 +101,9 @@
     };
   \end{cppcode}
   \pause
-  \begin{exampleblock}{The rule of 3/5/0 (\cpp98/\cpp11 and newer) - {\color{blue!50!white} \href{https://en.cppreference.com/w/cpp/language/rule_of_three}{cppreference}}}
-    \begin{itemize}
-    \item if a class has a destructor, a copy/move constructor or a copy/move assignment operator, it should have all three/five. strive for having none by using RAII types as members.
-    \end{itemize}
-  \end{exampleblock}
+  \begin{goodpracticeWithShortcut}{The rule of 3/5 (\cpp98/\cpp11) - {\color{blue!50!white}{Rule of 3/5} \href{https://en.cppreference.com/w/cpp/language/rule_of_three}{cppreference}}}
+    if a class needs a custom destructor, a copy/move constructor or a copy/move assignment operator, it should have all three/five.
+  \end{goodpracticeWithShortcut}
```

Comment:
This breaks the good practice link (its title has become "I" and that I is missing from the content of the block. to be fixed.

Also I believe we will need to lower the font of the good practice slide, it starts to be too full, and 2 titles don't fit in a line (also to be fixed)

Reply:
Hmm, I think I just pasted something in the wrong place here. The `{Rule of 3/5}` should be the name of the shortcut.